### PR TITLE
Added logout button

### PR DESCRIPTION
**Pull request by JustFinee**
Added button to sidebar menu

### CODE REVIEW

File: src/layouts/Main/components/Sidebar/Sidebar.js [hunk 1]
```diff
@@ -65,6 +68,12 @@ const Sidebar = props => {
       title: 'Settings',
       href: '/settings',
       icon: <SettingsIcon />
+    },
+    {
+      title: 'LogOut',
+      href: '/sign-in',
+      icon: <InputIcon />,
+      onClick: 'logout'
```
Comment (sebapaszynski) [inline src/layouts/Main/components/Sidebar/Sidebar.js:82]: What do we need onCLick here?

File: src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js [hunk 2]
```diff
@@ -50,10 +54,32 @@ const CustomRouterLink = forwardRef((props, ref) => (
...
+    //   .then(obj => obj.json())
+    //   .then(setCurrentUserAction(null))
+    //   .catch(e => console.error('Error during logout ', e));
+    //
+    // localStorage.removeItem('jwtToken');
+    // history.push('/sign-in');
+    setCurrentUserAction({});
+    localStorage.removeItem('jwtToken');
+    console.log("jestem w wylogowaniu");
+    console.log(localStorage.getItem('jwtToken'));
```
Comment (sebapaszynski) [inline src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js]: Please delete console.log() and unnecessary comments.

File: src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js [hunk 3]
```diff
@@ -85,4 +113,7 @@ SidebarNav.propTypes = {
   pages: PropTypes.array.isRequired
 };
 
-export default SidebarNav;
+export default connect(null, {
+  setCurrentUserAction: setCurrentUser,
+  getCurrentUserAction: getCurrentUser
```
Comment (sebapaszynski) [inline src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js]: We dont need getCurrentUserAction, please delete this action.

File: src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js [hunk 4]
```diff
@@ -85,4 +113,7 @@ SidebarNav.propTypes = {
   pages: PropTypes.array.isRequired
 };
 
-export default SidebarNav;
+export default connect(null, {
+  setCurrentUserAction: setCurrentUser,
```
Comment (sebapaszynski) [inline src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js:111]: setCurrentUserAction is missing in props validation (propTypes)

File: src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js [hunk 5]
```diff
@@ -50,10 +54,32 @@ const CustomRouterLink = forwardRef((props, ref) => (
...
 
 const SidebarNav = props => {
-  const { pages, className, ...rest } = props;
+  const { pages, className, getCurrentUserAction,
+    setCurrentUserAction, ...rest } = props;
 
   const classes = useStyles();
 
+  const handleSignOut = event => {
+    console.log(localStorage.getItem('jwtToken'));
```
Comment (sebapaszynski) [inline src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js]: Delete this console.log()

File: src/layouts/Main/components/Sidebar/Sidebar.js (hunk 1 above)
Comment (sebapaszynski) [reply src/layouts/Main/components/Sidebar/Sidebar.js:82]: No matter, I get it now - It's ok

File: src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js [hunk 6]
```diff
@@ -70,13 +96,15 @@ const SidebarNav = props => {
             className={classes.button}
             component={CustomRouterLink}
             to={page.href}
+            onClick={page.onClick == 'logout' ? handleSignOut : undefined}
```
Comment (sebapaszynski) [inline src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js]: Comparison page.onClick == 'logout' may cause unexpected type coercion. User triple equals operator ( === ) instead

File: src/layouts/Main/components/Sidebar/Sidebar.js [hunk 7]
```diff
@@ -9,6 +9,8 @@ import PeopleIcon from '@material-ui/icons/People';
 import ShoppingBasketIcon from '@material-ui/icons/ShoppingBasket';
 import AccountBoxIcon from '@material-ui/icons/AccountBox';
 import SettingsIcon from '@material-ui/icons/Settings';
+import { IconButton } from '@material-ui/core';
```
Comment (sebapaszynski) [inline src/layouts/Main/components/Sidebar/Sidebar.js]: 'IconButton' is defined but never used. Please delete this

File: src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js (hunk 2 above)
Comment (JustFinee) [reply src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js]: Done

File: src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js (hunk 3 above)
Comment (JustFinee) [reply src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js]: Done

File: src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js (hunk 4 above)
Comment (JustFinee) [reply src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js:111]: Done ( I guess)

File: src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js (hunk 5 above)
Comment (JustFinee) [reply src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js]: Done

File: src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js (hunk 6 above)
Comment (JustFinee) [reply src/layouts/Main/components/Sidebar/components/SidebarNav/SidebarNav.js]: Done

File: src/layouts/Main/components/Sidebar/Sidebar.js (hunk 7 above)
Comment (JustFinee) [reply src/layouts/Main/components/Sidebar/Sidebar.js]: Done

**Review by sebapaszynski — APPROVED** [review]

**Review by sebapaszynski — APPROVED** [review]

**Pull request merged**